web/servlet/home/Home_de.html

```diff
@@ -66,6 +66,12 @@
                                 <!-- should have open/download button as appropriate -->
                                 <a class="btn btn-default" href="http://www.withrottle.com" role="button">Erzähle weiter »</a>
                             </p>
+                            <h4><a href="https://www.microsoft.com/store/apps/9NBLGGH4VF5J">Train Throttle</a></h4>
+                            <p><em>Train Throttle</em> is een native gashendel voor de Windows. Het is geoptimaliseerd voor tablets en mobiel.</p>
```

Comment:
It's not nessassry for this to be rejected. Make the changes in your fork and commit them; this PR will automatically be updated with those changes.